Dockerfile-sshd

```diff
@@ -0,0 +1,22 @@
+#
+# To login to the SSH server, put a public key (id_rsa.pub) in /root/.ssh/authorized_keys
+#
+FROM alpine:3.5
+
+RUN apk --update add --no-cache openssh bash curl
+RUN sed -ie 's/#Port 22/Port 22/g' /etc/ssh/sshd_config
```

Comment:
Instead of doing sed, just commit the sshd_config and ADD file?

Reply:
Hehe right, I grabbed this Dockerfile from somewhere else and made some changes, but having the actual sshd file is simpler